.eslintrc.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
 		'wpcalypso/react',
 		'plugin:jsx-a11y/recommended',
 		'plugin:jest/recommended',
-		'prettier',
+		'plugin:prettier/recommended',
```

Comment:
Using the recommended config from https://prettier.io/docs/en/integrating-with-linters.html#recommended-configuration

This does three things:
- Extends `"prettier"` (as before)
- Add plugin `"prettier"` that allow running prettier as an eslint rule
- Configure prettier rules to use "error" level.